src/localization/messages_da.js

```diff
@@ -30,5 +30,7 @@ $.extend( $.validator.messages, {
 	maxWords:  $.validator.format( "Indtast venligst højst {0} ord" ),
 	step: $.validator.format( "Angiv en værdi gange {0}." ),
 	notEqualTo: "Angiv en anden værdi, værdierne må ikke være det samme.",
-	integer: "Angiv et ikke-decimaltal, der er positivt eller negativt."
+	integer: "Angiv et ikke-decimaltal, der er positivt eller negativt.",
```

Comment:
don't know why GitHub say this are re-added

Reply:
I guess it should detect and show that the only change was the comma at the end, but instead for some reason it got parsed as an entire block together with the next 2 lines...